### PR TITLE
Make the language in README zh_CN more native

### PR DESCRIPTION
**Pull request by FrozenArcher**
As a native Chinese speaker, I found the language in the original Chinese README not so readable, so I tried to fix it.
(my first pull request in my life QAQ)

### CODE REVIEW

**Comment by emersion** [comment]
Thanks for your PR! It would be nice if another Chinese speaker could review it.

cc @SuccessGo @3TUSK

File: README.zh-CN.md [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-irc.libera.chat).
+sway 是和 [i3](https://i3wm.org/) 兼容的 [Wayland](http://wayland.freedesktop.org/) compositor。
+ [阅读FAQ](https://github.com/swaywm/sway/wiki)/ [加入IRC频道](https://web.libera.chat/gamja/?channels=#sway) (#sway on irc.libera.chat)
 
-## 发布签名
+## 发行签名
 
-发布是以 [E88F5E48](https://keys.openpgp.org/search?q=34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48) 签名
-并发布在 [GitHub](https://github.com/swaywm/sway/releases).
+发行以 [E88F5E48](https://keys.openpgp.org/search?q=34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48) 签名并发布在 [GitHub](https://github.com/swaywm/sway/releases)上。
```
Comment (3TUSK) [inline README.zh-CN.md:8]:
```suggestion
每个发布版本都以 [E88F5E48](https://keys.openpgp.org/search?q=34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48) 的密钥签名并发布在 [GitHub](https://github.com/swaywm/sway/releases)上。
```

**Review by 3TUSK — CHANGES_REQUESTED** [review]
I was summoned. (By the way how did you know me?)

**Comment by emersion** [comment]
> By the way how did you know me?

You were involved in the pull request adding the Chinese translation a few years back. :)

**Comment by FrozenArcher** [comment]
改的好 -_-b

**Comment by FrozenArcher** [comment]
There's one more question: there're differences between English README and the Chinese one.

**Comment by emersion** [comment]
Yeah, translated READMEs often lag behind the English one. Would it be possible to sync up the Chinese README?

**Comment by FrozenArcher** [comment]
> Yeah, translated READMEs often lag behind the English one. Would it be possible to sync up the Chinese README? (Can be a separate PR if you prefer to get the current changes landed first.)

That's fine. But I'm only available on weekends. I'll do that this weekend.

**Comment by FrozenArcher** [comment]
I think the zh_TW README is translated very well.

**Comment by FrozenArcher** [comment]
> Yeah, translated READMEs often lag behind the English one. Would it be possible to sync up the Chinese README? (Can be a separate PR if you prefer to get the current changes landed first.)

I've already created a new one. This will be closed.

**Pull request closed without merging**